### ISSUE TITLE
404 when using the back button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#56]
```
What steps will reproduce the problem?
1. Log-in as admin
2. Use the back command on your browser
3. Click the log-in button or the link to voter's login

What is the expected output? What do you see instead?
I suppose that it should just redirect to the admin homepage since the
admin hasn't logged out yet, but what appears is this message:

404 Page Not Found
The page you requested was not found.
If you haven't installed Halalan yet, please install it now.

I tested this on Swiftweasel 3.0, Konqueror 3.5.9 on KDE 3.5.9 on Kubuntu 8.04.


```

Original issue reported on code.google.com by `toni...@gmail.com` on 27 Aug 2008 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
I can't reproduced this.  It successfully redirects to the admin home.
```

Original comment by `waldemar...@gmail.com` on 27 Aug 2008 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
i tested it on the demo... it also display a 404 page.
```

Original comment by `caket...@gmail.com` on 28 Aug 2008 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed this in r480.
```

Original comment by `toni...@gmail.com` on 29 Aug 2008 at 9:38
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision cd18a66da1.
```

Original comment by `toni...@gmail.com` on 4 Sep 2010 at 1:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `waldemar...@gmail.com` on 7 Feb 2011 at 9:17
- Changed state: **Verified**
